netket/operator/_continuous_operator.py

```diff
@@ -0,0 +1,98 @@
+# Copyright 2021 The NetKet Authors - All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import abc
+from typing import Optional, Callable
+
+from netket.utils.types import DType, PyTree, Array
+
+from netket.hilbert import AbstractHilbert
+from netket.operator import AbstractOperator
+
+
+class ContinousOperator(AbstractOperator):
+    r"""This class is the abstract base class for operators defined on a
+    continuous Hilbert space. Users interested in implementing new
+    quantum Operators for continuous Hilbert spaces should subclass
+    `ContinuousOperator` and implement its interface.
+    """
+
+    def __init__(self, hilbert: AbstractHilbert, dtype: Optional[DType] = float):
+        r"""
```

Comment:
```suggestion
    def __init__(self, hilbert: AbstractHilbert, dtype: DType = float):
        r"""
```
this should not be optional

Reply:
if there is a default it is optional or not?